ChaosMod/EffectDispatcher.cpp

```diff
@@ -86,10 +86,18 @@ void EffectDispatcher::UpdateTimer()
 		delta = 0;
 	}
 
-	if ((m_percentage = (delta + (m_timerTimerRuns * 1000)) / (m_effectSpawnTime * 1000)) > 1.f && m_dispatchEffectsOnTimer)
+	if ((m_percentage = (delta + (m_timerTimerRuns * 1000)) / (m_effectSpawnTime / g_metaInfo.timerSpeedModifier * 1000)) > 1.f && m_dispatchEffectsOnTimer)

```

Comment:
I would imagine a modifier should be a multiplier, rather than something to divide by, no? Represent 2x speed by a 50% duration modifier (i.e, `time * 0.5f`)?

Reply:
I don’t really get what you‘re saying. 
If it would be „multiplier“ then I would agree it could be assumed to be multiplied, but as a modifier...? 
i could make it (1/modifier), which would be the same 😅